pages/linux/yetris.md

```diff
@@ -0,0 +1,40 @@
+# yetris
+
+> Tetris(r) game clone in terminal.
```

Comment:
I'm not sure what the `r` in brackets is for here? Also, perhaps this will read better?
```suggestion
> Clone of the game Tetris in the terminal.
```

Reply:
So the Tetris name and brand is trademarked by The Tetris Company. 

I was/am not sure how that will work out here or legality of the name usage, so added "(r)" to it, which I saw in another Tetris clone's Debian manpage [here](https://manpages.debian.org/buster/bastet/bastet.6.en.html).

Reply:
Ah, I see. I was not aware of that!